br/pkg/task/backup.go

```diff
@@ -800,6 +800,13 @@ func DefaultBackupConfig() BackupConfig {
 	if err != nil {
 		log.Panic("infallible operation failed.", zap.Error(err))
 	}
+
+	// Check if the checksum flag was set by the user
+	if !fs.Changed("checksum") {
```

Comment:
use `flagChecksum` instead

Reply:
oh right... my mistake

Reply:
I feel like having override in `DefineBackupFlags` is a bit weird since it's technically not `define`, might be a bit confusing